var/spack/repos/builtin/packages/votca/package.py

```diff
@@ -39,7 +39,9 @@ class Votca(CMakePackage):
     depends_on("expat")
     depends_on("fftw-api@3")
     depends_on("eigen@3.3:")
-    depends_on("boost")
+    depends_on("boost@1.71:+filesystem+system+regex+timer")
+    depends_on("boost@1.71:1.84+filesystem+system+regex+timer", when="@2024")
+    depends_on("boost@1.71:1.82+filesystem+system+regex+timer", when="@:2023")
```

Comment:
Can we separate the common requests on boost from the version constraints:
```suggestion
    depends_on("boost+filesystem+system+regex+timer")
    depends_on("boost@1.71:")
    depends_on("boost@:1.84", when="@2024")
    depends_on("boost@:1.82", when="@:2023")
```
?

Reply:
Sure. I had the same thought last night, but wasn't sure it'd work that way. Should have tested, but it was late.